dev_gems.rb.lock

```diff
@@ -112,4 +112,4 @@ DEPENDENCIES
   webrick (~> 1.6)
 
 BUNDLED WITH
-   2.2.3
+   2.3.0.dev
```

Comment:
This means dev bundler should be bundled with itself, is that ok?

Reply:
Yes, I think it's good to test development bundler as much as we can, so if we can use it to bundle our own development dependencies, it sounds nice to me.

The reason I added this was that I normally have dev bundler installed glocally, so everytime I run `rake setup` the lockfile changes to the dev version. By always using the dev version that no longer happens.

Reply:
Good to go then, @simi?